c2corg_ui/static/js/card.js

```diff
@@ -119,6 +125,15 @@ app.CardController.prototype.createURL = function() {
 
 
 /**
+ * Don't create any url on edit and add pages.
+ * @export
+ * @return {string | undefined}
```

Comment:
Probably copy-paste from the function just below, gone wrong :P thanks for spotting it